Index: libssh2-0.17/example/Makefile.am
===================================================================
--- libssh2-0.17.orig/example/Makefile.am	2007-01-24 05:05:15.000000000 +0600
+++ libssh2-0.17/example/Makefile.am	2007-08-24 22:47:07.000000000 +0700
@@ -1,2 +1,2 @@
 AUTOMAKE_OPTIONS = foreign nostdinc
-SUBDIRS = simple
+SUBDIRS =
Index: libssh2-0.17/example/Makefile.in
===================================================================
--- libssh2-0.17.orig/example/Makefile.in	2007-07-31 17:02:52.000000000 +0700
+++ libssh2-0.17/example/Makefile.in	2007-08-24 22:47:07.000000000 +0700
@@ -175,7 +175,7 @@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
 AUTOMAKE_OPTIONS = foreign nostdinc
-SUBDIRS = simple
+SUBDIRS =
 all: all-recursive
 
 .SUFFIXES:
